### PR TITLE
Feat/i cards

### PR DESCRIPTION
**Pull request by arranf**
Best to review commit by commit I think!

Includes:

* Bump deps
* Minor cleanup of .gitignore and eslint
* Rework the Scorer to make tests more readable
* Fix a subtle source of bugs where a card's name could be added twice
* Added cards for I

### CODE REVIEW

File: src/scorer.ts [hunk 1]
```diff
@@ -44,13 +55,91 @@ export class Scorer {
...
+
+  
+  public get total() {
+      return this._total;
+  }
+
+  public getCardScoreByIndex(cardIndex: number) : CardScore {
+    const cardScore = this.playerCardsScores.at(cardIndex);
+    if (!cardScore) {
+      throw new Error(`Player of index ${cardIndex} not found`)
```
Comment (dwhenry) [inline src/scorer.ts]: **Card** of index

File: src/scorer.ts [hunk 2]
```diff
@@ -44,13 +55,91 @@ export class Scorer {
...
+      return new PlayerScore(playerCardsScores)
     });
 
-    return result;
+    const winningPlayersIndices = playerScores.reduce((maxScorePlayerIndices: number[], playerScore, index, arr) => {
+      const currentMax = arr[maxScorePlayerIndices[0]] // Get value of first max, but we can have multiple "max" values
+      if (playerScore.total === currentMax.total ) {
+        return [...maxScorePlayerIndices, index]
+      } else if (playerScore.total > currentMax.total) {
+        return [index]
```
Comment (dwhenry) [inline src/scorer.ts:75]: shouldn't this also update the `currentMax` value??

File: tests/b-cards.test.ts [hunk 3]
```diff
@@ -1,109 +1,60 @@
-import { Scorer } from '../src/scorer';
+import { Player, Scorer } from '../src/scorer';
 import { zeroPointColourlessCard, zeroPointGreenCard } from './helpers';
 
-describe('Using BIOENGINEERED PLAGUE + other cards', () => {
```
Comment (dwhenry) [inline tests/b-cards.test.ts:4]: this is a catastophe... can it go in a catastophe.test file.. :)

**Review by dwhenry — COMMENTED** [review]
small issues - but looks like a real improvement..

NOTE: we should add some scorer specific tests for determining the winning indicies

File: src/scorer.ts (hunk 1 above)
Comment (arranf) [reply src/scorer.ts]: Fixed

File: src/scorer.ts (hunk 2 above)
Comment (arranf) [reply src/scorer.ts:75]: No. The currentMax is calculated for each element in the array - no need to track the max.

File: tests/b-cards.test.ts (hunk 3 above)
Comment (arranf) [reply tests/b-cards.test.ts:4]: It was moved!

File: src/scorer.ts (hunk 2 above)
Comment (dwhenry) [reply src/scorer.ts:75]: this has confussed me... but I will believe you that it works...

**Review by dwhenry — APPROVED** [review]

**Pull request merged**